lib/src/common/orgs_service.dart

```diff
@@ -167,6 +175,29 @@ class OrganizationsService extends Service {
     return completer.future;
   }
 
+  /// Returns the membership status for a user in a team given the [orgName] and [teamName].
+  Future<TeamMembershipState> getTeamMembershipByName(
+      String orgName, String teamName, String user) {
+    final completer = Completer<TeamMembershipState>();
+
+    github
+        .getJSON(
+          '/orgs/$orgName/teams/$teamName/memberships/$user',
+          statusCode: 200,
+          fail: (http.Response response) {
+            if (response.statusCode == 404) {
+              completer.complete(TeamMembershipState(null));
```

Comment:
Is this necessary? I would expect callers of the library would want to handle this directly, instead of us returning null.

Reply:
It is not and was removed.